ios/FluentUI/People Picker/ContactCollectionViewCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class ContactCollectionViewCell: UICollectionViewCell {
+    static var identifier: String { return "ContactCollectionViewCell" }
+    var contactView: ContactView
+
+    override init(frame: CGRect) {
+        contactView = ContactView(title: "", subtitle: "")
+        super.init(frame: frame)
+    }
+
+    func setup(contact persona: PersonaData) {
+        if persona.name != "" {
+            contactView = ContactView(identifier: persona.name)
+        } else {
+            contactView = ContactView(identifier: persona.email)
+        }
```

Comment:
nit:
contactView = Contactview(identifier: (persona.name.length > 0) ? persona.name : persona.email))

Reply:
Didn't know you could use a ternary operator as an argument 😮 . Will try that approach!